ui/app/models/totp-key.js

```diff
@@ -101,26 +135,31 @@ export default class TotpKeyModel extends Model {
   skew;
 
   @attr('boolean', {
+    editType: 'toggleButton',
     defaultValue: true,
+    helperTextDisabled: 'Vault will not return QR code and url upon key creation.',
+    helperTextEnabled: 'QR code and URL will be returned upon generating a key.',
   })
   exported;
 
-  // Doesn't really make sense as we can generate our own QR code from the url
   @attr('number', {
+    label: 'QR size',
     defaultValue: 0,
   })
-  qr_size;
+  qrSize;
 
   // Used when generate is false
   @attr('string', {
     label: 'URL',
     helpText:
       'If a URL is provided the other fields can be left empty. E.g. otpauth://totp/Vault:test@test.com?secret=Y64VEVMBTSXCYIWRSHRNDZW62MPGVU2G&issuer=Vault',
+    subText: `The TOTP key url string that can be used to configure a key.`,
```

Comment:
total nit but these can be single quotes since there's no apostrophes in the string
```suggestion
    subText: 'The TOTP key url string that can be used to configure a key.',
```